src/index.scss

```diff
@@ -1,32 +1,7 @@
-// index.scss is for CSS declarations that aren't specific to a component.
-// In practice, that means this file is used for two things:
-//
-//   1. Setting the style of the <body> element
-//   2. Loading webfonts
+$dark-gray: #404040;
 
 body {
   margin: 0;
   padding: 0;
+  background-color: $dark-gray !important;
 }
-
-// Bold Button
-//
-// Use this class for a bolder, stronger looking button.
-//
-// Markup:
-// <button class="btn btn-bold">Click Me</button>
-//
-// Styleguide Components.Buttons.bold
-.btn.btn-bold {
-  position: relative;
-  font-weight: bold;
-  text-transform: uppercase;
-}
```

Comment:
This can't be removed until we add in some kss styleguide documentation elsewhere. Otherwise kss will fail on generation.